lib/api/request/kuzzleRequest.ts

```diff
@@ -196,19 +201,22 @@ export class KuzzleRequest {
   /**
    * Adds an error to the request, and sets the request's status to the error one.
    */
-  setError (error: Error) {
-    if (! error || !(error instanceof Error)) {
-      throw new InternalError('Cannot set non-error object as a request\'s error');
+  setError(error: Error) {
+    if (!error || !(error instanceof Error)) {
+      throw new InternalError(
+        'Cannot set non-error object as a request\'s error'
+      );
```

Comment:
Why reformating this? Even if it was slightly more than 80 column, it's better to keep it on a single line than to waste 2 lines